processors/cleanup_collections.py

```
@@ -0,0 +1,105 @@
+import sys
+import logging
+from arango import ArangoClient
+
+logging.basicConfig(
+    level=logging.INFO,
+    format='%(asctime)s - %(levelname)s - %(message)s'
+)
+logger = logging.getLogger(__name__)
+
+
+def cleanup_collections(keep_collections=['base_arxiv'], auto_confirm=False):
```

Ruff (0.12.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)